resources/views/products/edit.blade.php

```diff
@@ -0,0 +1,158 @@
+                    {{ 'Mark-It' }}
+                </div>
+                    <h2 class="page-title">
+                        {{ __('Edit Product') }}
+                    </h2>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="page-body">
+        <div class="container-xl">
+
+            @if ($message = Session::get('success'))
+            <div class="alert alert-success alert-dismissible">
+                <div class="d-flex">
+                    <div>
+                        <svg xmlns="http://www.w3.org/2000/svg" class="icon icon alert-icon" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
+                            <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+                            <path d="M5 12l5 5l10 -10"></path>
+                        </svg>
+                    </div>
+                    <div>
+                        {{ $message }}
+                    </div>
+                  </div>
+                <a class="btn-close" data-bs-dismiss="alert" aria-label="close"></a>
+            </div>
+            @endif
+
+            <form action="{{ route('products.store') }}" method="POST" class="card" autocomplete="off" enctype="multipart/form-data">
+                @csrf
+                <div class="card-body">
+
+
+                    <div class="mb-3">
+                        <label class="form-label required">{{ __('Name') }}</label>
+                        <input type="text" name="name" class="form-control @error('name') is-invalid @enderror" placeholder="{{ __('Name') }}" value="{{ old('name', $name['en']) }}" required>
+                    </div>
+                    @error('name')
+                    <div class="invalid-feedback">{{ $message }}</div>
+                    @enderror
+
+                    <div class="mb-3">
+                        <label class="form-label required">{{ __('Price') }}</label>
+                        <input type="number" name="price" class="form-control @error('price') is-invalid @enderror" placeholder="{{ __('price') }}" value="{{ old('price', $product->price) }}" required>
+                    </div>
+                    @error('price')
+                    <div class="invalid-feedback">{{ $message }}</div>
+                    @enderror
+
+                    <div class="mb-3">
+                        <label class="form-label required">{{ __('Quantity') }}</label>
+                        <input type="quantity" name="quantity" class="form-control @error('quantity') is-invalid @enderror" placeholder="{{ __('New quantity') }}" value="{{ old('quantity', $product->quantity) }}" required>
+                    </div>
+                    @error('quantity')
+                    <div class="invalid-feedback">{{ $message }}</div>
+                    @enderror
+
+                    {{-- <div class="mb-3">
```

Comment:
please don't comment them and at least send an empty array from the controller (or an array with one item similar to what the categories and the other items will be)